.travis.yml

```diff
@@ -18,7 +18,28 @@ env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
-  matrix:
+
+branches:
+  only:
+    - master
+    # npm version tags
+    - /^v\d+\.\d+\.\d+/
```

Comment:
These lines are what is preventing the build from running now. It's only running against `master` and npm version tags now. Can we also run against `develop`? Or just run everywhere by default?

Reply:
thanks for taking a look. Didn't think about that this might be related to the .travis config :facepalm's